### PR TITLE
Docs: Add an example to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nik9000**
Adds an example of accessing Elasticsearch after building it from source
for the first time to CONTRIBUTING.md. I'm sure how to do it is
intuitive to some folks but I had to read three two build.gradle file to
find the password and I'd like to save other new folks the trouble.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -119,6 +119,10 @@ For IntelliJ, the minimum version that we support is [IntelliJ 2017.2][intellij]
 
     ./gradlew :run
 
+You can access Elasticsearch with
+
+    curl elastic:password@localhost:9200
```
Comment (rjernst) [inline CONTRIBUTING.md]: Should we use `-u`?

**Review by rjernst — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -119,6 +119,10 @@ For IntelliJ, the minimum version that we support is [IntelliJ 2017.2][intellij]
 
     ./gradlew :run
 
+You can access Elasticsearch with
```
[the remark this answers is not in the document]
Comment (nik9000) [reply CONTRIBUTING.md]: Sure!

File: CONTRIBUTING.md (hunk 1 above)
Comment (nik9000) [reply CONTRIBUTING.md]: I'm happy to switch it.

**Comment by nik9000** [comment]
@elasticmachine, test this please

**Comment by nik9000** [comment]
Don't be lazy, my robot army!

**Pull request merged**